### ISSUE TITLE
Limited number of Sections in Main Menu

### ISSUE THREAD

**Issue by NavidNikbin** [Unity-Technologies/EditorVR#157]
Suppose that a solution needs a set of Tools and Workspaces. In my opinion putting all of them together in a new section of the Main Menu may be better than scattering them in Tools, Workspaces, and Create sections of the Main Menu. But the problem is the limited number of "Sections" in the Main Menu.

Some solutions:

1. While the physical shape of the Main Menu restrict it to having only 4 sections, It can logically have more sections which only 4 of them are visible at the same time. By rotating the Main Menu we can actually move between all visible and hidden sections (Like a paper roll)
2. Having a "Settings" button which shows all of the available sections and the user can select 4 desirable sections.
3. Having "Section Sets" that users can define them and move between them for example by pressing Up and Down buttons of "Thumb Pad"

Do you have any plan for removing this restriction?

**Comment by De-Panther** [comment]
I like the first option, but maybe the first section that show when opening the Main Menu should be the last one you opened.

**Comment by amirebrahimi** [comment]
Yes, @NavidNikbin, the original spec was supposed to have unlimited sections in the main menu that you could continue to roll through similar to what you describe as a "paper roll". This has not been implemented yet. I like the other options you provided, too.
